userguide/content/en/docs/Adding content/diagrams-and-formulae.md

```diff
@@ -0,0 +1,333 @@
+---
+title: "Diagrams and Formulae"
+linkTitle: "Diagrams and Formulae"
+weight: 11
+description: >
+  Add generated diagrams and scientific formulae to your site.
```

Comment:
Nit. Consider using the following simplified but equivalent front matter:

```yaml
title: Diagrams and Formulae
weight: 11
description: Add generated diagrams and scientific formulae to your site.
```

Reply:
Done.